src/components/Root.js

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import ReactGa from 'react-ga';
```

Comment:
Use react-ga4 instead ? Or we do not want to track app analytics with Google ?

Reply:
We don't use it anyway, so we'll have to change later.